source/actions/random-desktop.js

```diff
@@ -4,26 +4,35 @@
   getFiles,
   taskExecutor,
   filterImages,
+  validateOutput,
+  printJsonOutput,
 } from '../helpers';
 import setWallpaper from '../helpers/set-wallpaper';
 import {
   ORIENTATION_LANDSCAPE,
   ERROR_CODES,
 } from '../constants';
-import waitKeyToExit from '../helpers/wait-key-to-exit';
 
 /* Action that randomly set extracted images as desktop background */
 export default async function (args, options, logger) {
   // eslint-disable-next-line no-param-reassign
-  logger = logger.child({ caller: 'actions:random-desktop' });
+  logger = logger.child({caller: 'actions:random-desktop'});
+  const {output} = options;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5102014)